cmd/kubelet/app/options/options.go

```diff
@@ -232,6 +236,7 @@ func NewKubeletFlags() *KubeletFlags {
 		HostNetworkSources:  []string{kubetypes.AllSource},
 		HostPIDSources:      []string{kubetypes.AllSource},
 		HostIPCSources:      []string{kubetypes.AllSource},
+		CAdvisorPort:        4194,
```

Comment:
Can we add a TODO to change this to 0 in order to disable cAdvisor listening on the public port in the future?

Reply:
added